dts/riscv/ite/it8xxx2.dtsi

```diff
@@ -1007,13 +1013,19 @@
 			scl-gpios = <&gpioa 4 0>;
 			sda-gpios = <&gpioa 5 0>;
 			clock-gate-offset = <CGC_OFFSET_SMBF>;
+			clocks = <&ecpm IT8XXX2_ECPM_CGCTRL4R BIT(7) 0>;
 		};
 
 		ecpm: clock-controller@f01e00 {
 			compatible = "ite,it8xxx2-ecpm";
-			reg = <0x00f01e00 0x11>;
+			reg = <0x00f01e00 0x16>;
 			reg-names = "ecpm";
+			label = "EC_PM";
```

Comment:
remove label property

Reply:
Got it, sir. The label property has been removed.

Reply:
@galak 

Thanks again for guidance and revisiting.
If there's any further idea, please kindly let me know.